src/transformers/modeling_flax_pytorch_utils.py

```diff
@@ -92,7 +92,8 @@ def convert_pytorch_state_dict_to_flax(pt_state_dict, flax_model):
             pt_tuple_key = pt_tuple_key[:-1] + ("embedding",)
         elif pt_tuple_key[-1] == "weight" and pt_tuple_key not in random_flax_state_dict:
             pt_tuple_key = pt_tuple_key[:-1] + ("kernel",)
-            pt_tensor = pt_tensor.T
+            is_conv = pt_tensor.ndim == 4  # hackish way of detecting conv layer
```

Comment:
Can we get something from the name for this?

Reply:
Ah that's unfortunate that we can't really do much here except matching by `.ndim` :-/ since both Conv and Linear are called `weight` in PT and `kernel` in Flax. 

Could we make it a bit more readable though by adding a new elif:

```python
elif pt_tuple_key[-1] == "weight" and and pt_tensor.ndim == 4 and pt_tuple_key not in random_flax_state_dict:
# conv layer
   ....
elif pt_tuple_key[-1] == "weight" and pt_tuple_key not in random_flax_state_dict:
# dense layer
   ....